package.json

```diff
@@ -193,7 +193,7 @@
     "ansi-regex": "^5.0.1",
     "antlr4ts": "^0.5.0-alpha.3",
     "archiver": "^5.3.1",
-    "axios": "^1.6.0",
+    "axios": "1.6.3",
```

Comment:
Any reason to drop the ^?

Reply:
ahh good catch. added back in latest commit